backend/barber_shop/management/views.py

```
@@ -264,4 +264,27 @@ def get_attendance_log(request):
+    serializer = ClientSerializer(client)
+    return Response(serializer.data, status=200)
+
+
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def get_barber_by_id(request, barber_id):
+    """
+    Devuelve la información de un barbero por su ID.
+    """
+    barber = get_object_or_404(BarberProfile, id=barber_id)
+    serializer = BarberProfileSerializer(barber)
```

Ruff (0.7.0)

`BarberProfileSerializer` may be undefined, or defined from star imports

(F405)